packages/http-user-agent/.eslintrc.js

```diff
@@ -0,0 +1,41 @@
+module.exports = {
+  parser: '@typescript-eslint/parser',
```

Comment:
As mentioned before, for now TS packages should have their own eslint config, which is merged with the base one.

Once we've migrated all packages, we can merge this back with the base one.